content/pages/legal.md

```diff
@@ -1,172 +1,161 @@
-letter from Social Security in addition to your court order. While you're at
-the DMV, you should also update your car registration and title.
+24 hours after updating with Social Security, you can update your driver's
+license.
 
-Once you've updated your driver's license you should start updating all the
-other places that know your name. Some important places to notify:
+- Most DMVs should be able to change your name.
+- Be sure to bring your letter from Social Security in addition to your original court order.
+- You should also update your voter registration
+  information and car registration/title if applicable.
 
- - Your employer's HR department
- - Banks
- - Credit card companies
- - And many more (it's amazing how many people know your name!)
+Once you've got a new ID, you can start elsewhere. Some important places:
+
+- Your employer's HR department
+- Bank and credit card companies
+- Your car insurance company
+- Your health insurance company
+- Your school or university
+- Doctors and/or medical providers
+- Your local post office (setup mail forwarding!)
+- And many more (it's amazing how many people know your name).
 
 # Updating your gender marker
 
-Currently, having an updated birth certificate is the quickest way to change
-your gender marker on other legal docs (like your driver's license). If you're at 
-least on hormones and can get a letter from a doctor or other provider, this 
-process may be possible for you.
+Currently, having an updated birth certificate is the quickest way to change your gender marker on other legal documents.
 
-At this time, Alabama does not offer a nonbinary gender identification.
+- If you're at least on hormones and can get a letter from a doctor or other provider, this process may be possible.
+- As of January 2025, Alabama does not offer a nonbinary or third gender identification option.
 
 ## Birth Certificate
 
-Following the letter of the law in Alabama, you must obtain a letter of
-transition from a medical provider to update the gender on your birth
-certificate.
+You **must** obtain a letter of transition from a medical provider to update the gender on your birth certificate.
+
+- If you have an Alabama birth certificate, [a guide is provided by ALTRAC to update it without a lawyer][altrac-genderguide].
+- This has been proven to work in Madison County, and **could** potentially work in other counties.
+
+Morgan County will not process birth certificate changes.
+
+- If you know of any other counties that claim to not have jurisdiction, please notify ALTRAC at [gender@altrac.works][email-altracgender].
 
-If you have an Alabama birth certificate, [you can use this guide from ALTRAC 
-to update it without the use of a lawyer][gender-guide]. This has been proven
-to work in Madison County, and could potentially work in other counties.
+If you were born outside Alabama, refer to [the specific guide for your
+state][transequality].
```

Comment:
I'm not sure I understand what is and is not turned into a bullet-point

Reply:
Split this up a little bit better just now.